test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7101,6 +7101,14 @@ def forward(self, fp16, fp32):
         fp32 = Tensor([1.5])
         self.run_test(CatModel(), (fp16, fp32))
 
+        class DoNotUpcastModel(torch.nn.Module):
```

Comment:
Can this be a separate test case? I would explain a little more on what it is testing in code.